src/main/java/com/vmtapp/enterprise/dao/PhotoDAO.java

```diff
@@ -0,0 +1,35 @@
+package com.vmtapp.enterprise.dao;
+
+import com.vmtapp.enterprise.dto.Photo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@Repository
+public class PhotoDAO implements IPhotoDAO {
+
+    @Autowired
+   // private PhotoRepository photoRepository;
+
+    @Override
+    public void save(Photo photo) {
+        //photoRepository.save(photo);
```

Comment:
This method name is not representative of what it does, which is misleading.  Only commit and push complete features to VCS.